composer.json

```diff
@@ -20,6 +21,6 @@
     },
     "require-dev": {
         "phpunit/phpunit": "~3.7.22",
-        "satooshi/php-coveralls": "^1.0"
+        "satooshi/php-coveralls": "^0.6.1"
```

Comment:
This is the version that is actually locked to. Running

```
$ composer update --lock
```

after db16087af99e435c4f6001490acef55fab1729c4 yields

```
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - The requested package satooshi/php-coveralls (locked at v0.6.1, required as ^1.0) is satisfiable by satooshi/php-coveralls[v0.6.1] but these conflict with your requirements or minimum-stability.
```

so I ran

```
$ composer require --dev satooshi/php-coveralls:^0.6.1
```